conf/README.md

```diff
@@ -41,21 +38,172 @@ All of the below sections, will be available from the ocsci config dataclass.
+* `force_chrome_branch_sha256sum` - Chrome branch sha256sum for openshift console UI testing
+* `chrome_binary_path` - Filepath to the chrome browser binary
+* `io_in_bg` - Run IO in background (Default: false)
+* `io_load` - Target percentage for IO in background
+* `log_utilization` - Enable logging of cluster utilization metrics every 10 seconds. Set via --log-cluster-utilization
+* `use_ocs_worker_for_scale` - Use OCS workers for scale testing (Default: false)
+* `load_status` - Current status of IO load
+
 #### DEPLOYMENT
 
 Deployment related parameters. Only deployment related params not used
 anywhere else.
 
+* `installer_version` - OCS installer version
+* `force_download_installer` - Download the OCS installer even if one already exists in the bin_dir
+* `force_download_client` - Download the OCP client even if one already exists in the bin_dir
+* `default_latest_tag` - OCS latest tag to be used by default if one is not provided
+* `external_mode` - If OCS cluster is setup in external mode (Default: false)
+* `ocs_csv_channel` - Channel used to install OCS CSV
+* `default_ocs_registry_image` - Default OCS registry image
+* `ocs_operator_nodes_to_label` - Number of OCS operator nodes to label
+* `ocs_operator_nodes_to_taint` - Number of OCS operator nodes to taint
+* `ssh_key` - Filepath to the public SSH key used to authenticate with OCP nodes
+* `ssh_key_private` - Filepath to the private SSH key used to auth with OCP nodes
+* `force_deploy_multiple_clusters` - Allow multiple clusters to be deployed with the same prefix (vmware)
+* `allow_lower_instance_requirements` Allow instance requirements lower than the documented recommended values (Default: false)
+* `ui_deployment` - Utilize openshift-console to deploy OCS via the UI (Default: false)
+* `live_deployment` - Deploy OCS from live content (Default: false)
+* `live_content_source` - Content source to use for live deployment
+* `preserve_bootstrap_node` - Preserve the bootstrap node rather than deleting it after deployment (Default: false)
+* `terraform_version` - Version of terraform to download
+* `infra_nodes` - Add infrastructure nodes to the cluster
+* `openshift_install_timeout` - Time (in seconds) to wait before timing out during OCP installation
+* `local_storage` - Deploy OCS with the local storage operator (Default: false)
+* `disconnected` - Set if the cluster is deployed in a disconnected environment
+* `mirror_registry` - Hostname of the mirror registry
+* `mirror_registry_user` - Username for disconnected cluster mirror registry
+* `mirror_registry_password` - Password for disconnected cluster mirror registry
+* `min_noobaa_endpoints` - Sets minimum noobaa endpoints (Workaround for https://github.com/red-hat-storage/ocs-ci/issues/2861)
+* `host_network` - Enable host network in the storage cluster CR and prepare rules needed in AWS for host network during OCP deployment
+* `subscription_plan_approval` - 'Manual' or 'Automatic' subscription approval for OCS upgrade
```

Comment:
Do you want to mention Default value for subscription_plan_approval ?

Reply:
We don't have a value for this in our default config